internal/util/validation/validation.go

```diff
@@ -85,6 +85,26 @@ func ValidateBase64Field(s string, path string, maxLen int) []error {
 	return asErrors(errs)
 }
 
+func ValidateBearerToken(token *string, path string) []error {
+	if token == nil {
+		return []error{}
+	}
+	var jwtPattern = regexp.MustCompile(`^[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+$`)
```

Comment:
this regex appears to be correct but can we reference the spec for JWT here?

- https://www.rfc-editor.org/info/rfc7519